css/values.css

```diff
@@ -0,0 +1,248 @@
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+
+/* need to add wrapper */
+/* red background for bottom section needs fixing */
+/* some text changes from white to black when in desktop mode */
+/* blue background for support us needs fixing */
+/* heart color and gap needed for support us */
+/* white icons in support us need flexbox to move side by side in desktop mode */
+/* align title text to center somehow hoping wrapper fixes some stuff */
+html {
+	/* does this need before and after */
+	box-sizing: border-box;
+}
+body {
+}
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+.hero {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+.card{
+	min-height: 204px;
+	color: var(--color-white);
+	border: none;
+	border-radius: 10px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin: 0px 20px;
+}
+.card h4 {
+	margin: 20px 0px 0px 0px;
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+}
+.card p {
+	display: flex;
+	text-align: left;
+	font-size: var(--font16);
+    font-weight: var(--font-weight-regular);
+    line-height: var(--line-height20);
+	margin: var(--spacing20);
+}
+/* Turns Our Values Title Black */
+.values-head {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	margin-bottom: 100px;
+}
+
+.h3-black {
+	color: var(--color-onyx);
+	margin: 60px 0 20px 0;
+}
+
+.values-head p {
+	color: var(--color-onyx);
+	margin: 0 20px 0;
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.copperback {
+	background-color: var(--color-copper);
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	margin-bottom: 20px;
+}
+
+/* Adds a background color in lieu of those rectangle shapes */
+
+.onyxback {
+	margin-top: 20px;
+	background-color: var(--color-onyx);
+}
+
+/* Adds a background color in lieu of those rectangle shapes */
+
+.amaraback {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	background-color: var(--color-amaranth);
+}
+
+/* Adds a background color in lieu of those rectangle shapes */
+
+.grapeback {
+	margin-top: 20px;
+	background-color: var(--color-cybergrape);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+
+.blueback {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	background-color: var(--color-yaleblue);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+
+.tealback {
+	margin-top: 20px;
+	background-color: var(--color-teal);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+
+.support {
```

Comment:
Spacing and text sizing on this section is off on desktop. Looks good on mobile though. 

<img width="926" alt="Screen Shot 2024-01-24 at 8 01 22 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/3ddb16c8-b9b2-4d90-92a2-1a0939877580">